carWarGame.py

```diff
@@ -1,6 +1,8 @@
+import random
+
 class Card:
 
-    values = {'2': 2, '3': 3, '4': 4, '5': 5, '6': 6, '7': 7, '8': 8, '9': 9, '10': 10, 'J': 11, 'Q': 12, 'K':13, 'A':14}
+    values = {'2':2,'3':3,'4':4,'5':5,'6':6,'7':7,'8':8,'9':9,'10':10,'J':11,'Q':12,'K':13,'A':14}
```

Comment:
@andresgamo please investigate black and reformat your code, please.

https://black.readthedocs.io/en/stable/